### ISSUE TITLE
Feat: Add short description to servers:

### ISSUE THREAD

**Issue by MordechaiHadad** [neovim/nvim-lspconfig#1270]
### Requested feature

I think shirt description for each server should be added for example in clangd:
in docs {
short_description = "clangd is a language server, and provides C++ IDE features to editors."
}

**Comment by mjlbach** [comment]
We already do this under the documentation per server, if there are documentation issues please file PRs addressing them

**Comment by MordechaiHadad** [comment]
> 
> 
> We already do this under the documentation per server, if there are documentation issues please file PRs addressing them

True, but this is not what i'm asking add this to the actual [server].lua file in-case someone wants to access a short description

**Comment by mjlbach** [comment]
This is stored under the server.lua files, the documentation is auto generated from this.

**Comment by MordechaiHadad** [comment]
> 
> 
> This is stored under the server.lua files, the documentation is auto generated from this.

yes i know but this description is super long, what i meant is have one description that is short and one that is the long one, example:

```lua
docs {
    short_description = "clangd is a language server, and provides C++ IDE features to editors.",
    description = [[
    https://clangd.llvm.org/installation.html
   **NOTE:** Clang >= 9 is recommended! See [this issue for more](https://github.com/neovim/nvim-lsp/issues/23).
   clangd relies on a [JSON compilation database](https://clang.llvm.org/docs/JSONCompilationDatabase.html) specified
   as compile_commands.json or, for simpler projects, a compile_flags.txt.
   For details on how to automatically generate one using CMake look [here](https://cmake.org/cmake/help/latest/variable /CMAKE_EXPORT_COMPILE_COMMANDS.html).
]],
}
```

**Comment by mjlbach** [comment]
I see this as duplication, lspconfig has no internal use for this. You can inject whatever you would like in an installer plugin into the lspconfig data structure.

**Comment by MordechaiHadad** [comment]
> 
> 
> I see this as duplication, lspconfig has no internal use for this. You can inject whatever you would like in an installer plugin into the lspconfig data structure.

fair enough, thank u anyway.
